library/src/main/java/com/blarley/rxretrobusseed/library/bus/RxRetroBus.java

```diff
@@ -93,10 +93,16 @@ public void register(RetroSubscriberReceiver receiver) {
             if (cachedResponse != null) {
                 if (cachedResponse.isError()) {
                     postError(subscriber, cachedResponse.getError());
+                    if (cachedResponse.isSticky()) {
```

Comment:
This bothers me for whatever reason, change the logic to if(loading) else { if(sticky) {remove}; if (success) {post success } else if (error)  {post error}}